Persephone/Components/Browser/Album Detail/AlbumDetailView.swift

```diff
@@ -128,15 +128,18 @@ class AlbumDetailView: NSViewController {
         self.dataSource.albumSongs[1].song
         else { return }
 
-      self.getBigCoverArt(song: song, album: album)
+      DispatchQueue.main.async {
+        self.getBigCoverArt(song: song, album: album)
```

Comment:
self is implicit

Reply:
Yeah, I usually let Xcode tell me where I need `self` :)